apps/session-manager/src/utils/environment.ts

```diff
@@ -17,3 +20,14 @@ export function getRequiredENVVar(envName: string): string {
     return envVal;
   }
 }
+
+export const getENVVarWithDefault = <T>(
```

Comment:
Add a small function doc comment

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/22/commits/0aac1a7af5e32060657c60e53ef458e41e4e445c